sha3/src/main/java/com/theromus/sha/Keccak.java

```diff
@@ -105,7 +234,7 @@ private void doKeccakf(final int[] uState) {
      *
      * @param state state
      */
-    private void roundB(final BigInteger[][] state) {
+    private static void roundB(final BigInteger[][] state) {
```

Comment:
Convert to non-static.

Reply:
Why? isn't better leave this function static?